army schema work/1 table.sql

```diff
@@ -0,0 +1,42 @@
+use ArmyDB
+go
+
+drop table if exists dbo.Soldier
+go 
+
+create table dbo.Soldier
+(
+FirstName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_FirstName_cannot_be_blank CHECK (FirstName <> ''),
+LastName VARCHAR(20) not null
```

Comment:
-- LL the first and last name columns should have a slightly higher varchar allowance